src/components/home/ChooseTailcall.tsx

```diff
@@ -24,7 +29,24 @@ const ChooseTailcall = (): JSX.Element => {
           </div>
         ))}
       </div>
-    </>
+      <div className="flex py-10 gap-y-SPACE_04 gap-x-SPACE_04 md:gap-y-SPACE_06 md:gap-x-SPACE_06 flex-wrap md:items-center md:justify-center">
+        {tailcallFeatures.map((item) => (
+          <div
+            className="flex w-fit p-6 border border-solid border-tailCall-border-dark-300 rounded-3xl md:items-center md:justify-center cursor-pointer"
+            key={item.id}
+            onClick={() => handleClick(item.redirection_url)}
+          >
+            <img src={item.image} alt={`${item.title} Image`} height={24} width={24} />
+            <span className="text-content-tiny sm:text-content-small text-bold ml-2">{item.title}</span>
+          </div>
+        ))}
+      </div>
+      <LinkButton
+        theme={Theme.Gray}
+        onClick={() => handleClick("/docs/graphql-n-plus-one-problem-solved-tailcall/")}
+        title="View More"
```

Comment:
Let's take them to http://localhost:3000/docs/graphql-configuration-generation-with-tailcall/